osal/src/os/posix/inc/os-freertos.h

```diff
@@ -40,21 +40,16 @@
 #include <stdlib.h>
```

Comment:
This was removed from `bsp_start.c`, should it be removed here, or added back there?

Reply:
I think either is fine. My general reasoning was I diffed the posix directory with either vxworks/rtems (forget which) and if it still had the include I left it there. I figured at the team it was best to leave so people knew what libraries were available. But we can just strip out everything that still lets it compile.